libs/api/domains/university-of-iceland/src/lib/dto/studentInfo.input.ts

```diff
@@ -1,8 +1,8 @@
 import { Field, InputType } from '@nestjs/graphql'
 import { IsNumber, IsOptional, IsString } from 'class-validator'
 
-@InputType()
-export class UniversityOfIcelandStudentInfoQueryInput {
+@InputType('UniversityOfIcelandStudentInfoInput')
```

Comment:
Need to change the gql in `libs/service-portal/education/src/screens` as well. 
UniversityOfIcelandStudentInfoQueryInput -> UniversityOfIcelandStudentInfoInput